### ISSUE TITLE
Nesting alignment validator

### ISSUE THREAD

**Issue by mensfeld** [mensfeld/yard-lint#191]
we should offend on stuff like that

```ruby
# frozen_string_literal: true

class OverloadDocumented
  #   @param key [Symbol] the key
  #      @return [Object] the value
  def fetch(key)
    key
  end
```

**Comment by mensfeld** [comment]
unless `@overload` or other supported cases ofc
